.ci/Jenkinsfile

```diff
@@ -95,16 +94,19 @@ pipeline {
         PATH = "${env.HOME}/go/bin:${env.PATH}"
       }
       steps {
-        deleteDir()
-        unstash 'source'
-        dir("${BASE_DIR}"){
-          sh(label: 'Runs the (integration) tests',script: '''
-            eval "$(gvm 1.14.2)"
-            go get -u github.com/magefile/mage
-            go get -u github.com/elastic/go-licenser
-            go get -u -v golang.org/x/tools/cmd/goimports
-            mage -debug testIntegration
-          ''')
+        cleanup()
+        withMageEnv(){
+          dir("${BASE_DIR}"){
+            sh(label: 'Runs the (integration) tests',script: 'mage -debug testIntegration|tee tests-report.txt')
```

Comment:
Redirect stderr?

```suggestion
            sh(label: 'Runs the (integration) tests',script: 'mage -debug testIntegration 2>&1 | tee tests-report.txt')
```

Reply:
I have tested and there is nothing in the stderr even do some tests fail